doc/source/developer_overview.rst

```diff
@@ -120,7 +120,7 @@ Here a list of the |packages| in the repository along with a short description:
 
 -  ``ament_cmake_test``
 
-   - aggregates different kinds of tests, e.g. gtest and nosetests, under a single target using CTest
+   - aggregates different kinds of tests, e.g. gtest and nosetests, under a single target using `CTest <https://cmake.org/cmake/help/v3.6/manual/ctest.1.html>`_
```

Comment:
Yeah, it's not the same content that I was looking for, but I think the wiki page should be alright.